ports/stm32/sdram.c

```diff
@@ -255,7 +255,7 @@ static void sdram_init_seq(SDRAM_HandleTypeDef
     #endif
 }
 
-bool __attribute__((optimize("O0"))) sdram_test(bool fast) {
+bool sdram_test(bool fast) {
```

Comment:
Is there a reason to remove the optimize attribute?

Reply:
I couldn't see any reason why it'd need to be there anymore, especially at no-optimize, taking more space than it really should need to.